model/project_ref_test.go

```diff
@@ -3633,7 +3634,7 @@ func TestGetActivationTimeForVariant(t *testing.T) {
 	}
 	assert.Nil(version.Insert())
 
-	activationTime, err = projectRef.GetActivationTimeForVariant(&BuildVariant{Name: "bv"})
+	activationTime, err = projectRef.GetActivationTimeForVariant(&BuildVariant{Name: "bv"}, time.Now())
 	assert.NoError(err)
 	assert.NotZero(activationTime)
```

Comment:
Is this also supposed to check that the activation time is the time passed into `GetActivationTImeForVariant`?

Reply:
Actually sorry I changed this test without thinking too much about this -- the first instance of this test does check this;  the second build variant is disabled so the zero time is correct. Updating this test to be clearer.